extensions/data-transfer/portability-data-transfer-rememberthemilk/src/main/java/org/dataportabilityproject/transfer/rememberthemilk/tasks/RememberTheMilkTasksExporter.java

```diff
@@ -95,9 +101,17 @@ private ExportResult exportTask(
         for (TaskSeries taskSeries : taskList.taskseries) {
           // TODO: figure out what to do with notes
           String notesStr = taskSeries.notes == null ? "" : taskSeries.notes.toString();
-          tasks.add(new TaskModel(oldListId, taskSeries.name, notesStr));
           for (Task task : taskSeries.tasks) {
-            // TODO: handle completion date, but its odd there can be more than one.
+            // TODO: What do we actually want to do with this?
```

Comment:
Does this comment apply or does it have to be more specific?

Reply:
Clarified.